include/swift/AST/DiagnosticsSIL.def

```diff
@@ -177,6 +177,10 @@ ERROR(self_disallowed_actor_init,none,
       "actor 'self' %select{can only|cannot}0 %1 from "
       "%select{an async|a global-actor isolated}0 initializer",
       (bool, StringRef))
+NOTE(actor_convenience_init,none,
+     "convenience initializers allow non-isolated use of 'self' once "
+     "initialized",
```

Comment:
once fully initialized?

Reply:
We use "fully initialized" within the compiler comments, etc, but the diagnostics in `DiagnosticsSIL.def` we never say "fully" in a message, just in the name of the diagnostics 🤷🏼

Reply:
i see, got it thanks!